pkg/utils/util.go

```diff
@@ -60,12 +59,25 @@ const (
 
 	// [Deprecated] CSI Parameters that are put into fields but
 	// NOT stripped from the parameters passed to CreateSnapshot
-	snapshotterSecretNameKey      = "csiSnapshotterSecretName"
-	snapshotterSecretNamespaceKey = "csiSnapshotterSecretNamespace"
+	SnapshotterSecretNameKey      = "csiSnapshotterSecretName"
```

Comment:
Why change this to exported?

Reply:
util.go is used by both common and sidecar controller.  So this is no longer internal.

Reply:
Comment says that these keys are deprecated.  Can we remove support for it?

Reply:
this is still outstanding

Reply:
Done